.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,28 @@
+name: CodeQL
```

Comment:
Add CodeQL and the PR is called `Add documentation` 😅

Reply:
Yes :)
Didn't want to open a seperate pr for such a trivia.